generators/client/templates/angular/src/test/javascript/spec/app/account/activate/_activate.component.spec.ts

```diff
@@ -0,0 +1,96 @@
+import {ComponentFixture, TestBed, async, tick, fakeAsync} from '@angular/core/testing';
```

Comment:
follow the spacing in other files plz `import { ComponentFixture, TestBed, async, tick, fakeAsync } from '@angular/core/testing';` for all imports